Code Editor and React version

Trying to use Code Editor beta feature, and I'm having dependency issues.
 
- Run ```npm install @patternfly/react-code-editor --sav```

```bash
Could not resolve dependency:
npm ERR! peer react@"^17.x" from react-monaco-editor@0.41.2
npm ERR! node_modules/react-monaco-editor
npm ERR!   peer react-monaco-editor@"^0.41.2" from @patternfly/react-code-editor@4.2.18
npm ERR!   node_modules/@patternfly/react-code-editor
npm ERR!     @patternfly/react-code-editor@"*" from the root project
```

Looks like a dependency bug, react code editor asking for react 16 but the react monaco editor 17. and the latest version of patternfly also on 17.

I'm using the latest release of April

--- a/packages/react-code-editor/package.json
+++ b/packages/react-code-editor/package.json
@@ -39,8 +39,8 @@
   "peerDependencies": {
     "monaco-editor": "^0.21.3",
     "monaco-editor-webpack-plugin": "^2.1.0",
-    "react": "^16.8.0",
-    "react-dom": "^16.8.0",
+    "react": "^16.8.0 || ^17.0.0",
+    "react-dom": "^16.8.0 || ^17.0.0",
     "react-monaco-editor": "^0.41.2"
   },
   "devDependencies": {

